rust/src/bin/weekly_update_player_scores.rs

```diff
@@ -157,13 +148,14 @@ fn main() -> BoxResult<()> {
     println!("Loaded {} uuids in {} milliseconds", uuids.len(), (end - start).num_milliseconds());
 
     let start = Utc::now().time(); // START
+    let days_since_epoch: i32 = (Utc::now() - Utc.timestamp(0, 0)).num_days() as i32;
```

Comment:
In-game the date rolls over at midnight for GMT-9 rather than UTC/GMT; this should not matter, as instances are garbage collected a week after expiration, but it would be wise to add a comment for if/when we optimize this later.